pandas/core/arrays/datetimes.py

```diff
@@ -832,7 +832,14 @@ def normalize(self):
                        '2014-08-01 00:00:00+05:30'],
                        dtype='datetime64[ns, Asia/Calcutta]', freq=None)
         """
-        new_values = conversion.normalize_i8_timestamps(self.asi8, self.tz)
+        if self.tz is None:
+            not_null = self.notnull()
```

Comment:
same

Reply:
should this be `notna`?  (does `DatetimeArray` even _have_ notna or notnull?)

Reply:
It (`DatetimeIndex`) apparently has `notnull`, but not sure if i should be using `notna` or `notnull`

Reply:
can you use notna